### DISCUSSION TITLE
Customize

### DISCUSSION THREAD

**Discussion by DotHack88** [jef/streetmerchant#2880] [General]
Hello everyone, I would like to ask you if there is the possibility of being able to customize the reception message by telegram.
I would like to change it and give it a personal touch. It's possible ?

![immagine](https://user-images.githubusercontent.com/40268801/145564831-45e1b957-22c0-4baf-8da7-9f508794f259.png)

Thx lot men

**Comment by Mr-Tech-13** [comment]
Yes. Go to src/messaging/telegram.ts and open that up. Then you can customize it!

**Reply by DotHack88** [reply]
okay, when i've 2 second i try it
